src/test/java/org/folio/oaipmh/common/AbstractInstancesTest.java

```diff
@@ -0,0 +1,100 @@
+package org.folio.oaipmh.common;
+
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.junit5.VertxTestContext;
+import org.apache.commons.collections4.CollectionUtils;
+import org.folio.oaipmh.dao.InstancesDao;
+import org.folio.rest.jooq.tables.pojos.Instances;
+import org.folio.rest.jooq.tables.pojos.RequestMetadataLb;
+import org.jooq.JSON;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+import javax.ws.rs.NotFoundException;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static org.folio.rest.impl.OkapiMockServer.OAI_TEST_TENANT;
+
+public abstract class AbstractInstancesTest {
+
+  protected static final int INSTANCES_EXPIRATION_TIME_IN_SECONDS = 7200;
```

Comment:
so the request Id which was not called in last 2 hours is deleted from the instances table, what is the behaviour if they request the data with resumption token after 2 hours? How will the API respond?